spec/features/filling_in_the_form/fill_in_the_form_spec.rb

```diff
@@ -2,46 +2,58 @@
 
 require "spec_helper"
 
-RSpec.feature "Fill in the form" do
+RSpec.feature "fill in the business volunteer form" do
   include FillInTheFormSteps
 
-  scenario "Complete the form" do
-    given_a_business_during_the_covid_19_pandemic
-    that_can_offer_medical_equipment
-    and_is_a_manufacturer_a_distributor_and_agent
-    and_has_personal_protection_equipment_available
-    and_has_other_medical_equipment_available
-    and_can_offer_hotel_rooms
-    and_can_offer_transport_or_logistics
-    and_can_offer_space
-    and_can_offer_all_types_of_expertise
-    and_can_offer_all_kinds_of_social_and_child_care
-    and_offers_these_across_the_uk
-    and_has_given_their_business_details
-    and_has_supplied_a_contact
-    and_has_accepted_the_terms_and_conditions
-    then_they_are_thanked
+  shared_examples "completing the form" do
+    scenario "fill in the form" do
+      given_a_business_during_the_covid_19_pandemic
+      that_can_offer_medical_equipment
+      and_is_a_manufacturer_a_distributor_and_agent
+      and_has_personal_protection_equipment_available
+      and_has_other_medical_equipment_available
+      and_can_offer_hotel_rooms
+      and_can_offer_transport_or_logistics
+      and_can_offer_space
+      and_can_offer_all_types_of_expertise
+      and_can_offer_all_kinds_of_social_and_child_care
+      and_offers_these_across_the_uk
+      and_has_given_their_business_details
+      and_has_supplied_a_contact
+      and_has_accepted_the_terms_and_conditions
+      then_they_are_thanked
+    end
   end
 
-  scenario "Ensure we can perform a healthcheck" do
+  describe "fill in the form" do
+    context "without javascript" do
+      it_behaves_like "completing the form"
+    end
+
+    context "with javascript", js: true do
+      it_behaves_like "completing the form"
```

Comment:
Is this all that's required to run the specs with/without JS? If so, this is a neat way to do it 👍